src/spec-node/containerFeatures.ts

```diff
@@ -63,14 +63,17 @@ export async function extendImage(params: DockerResolverParameters, config: DevC
 	for (const buildArg in featureBuildInfo.buildArgs) {
 		args.push('--build-arg', `${buildArg}=${featureBuildInfo.buildArgs[buildArg]}`);
 	}
+	// Once this is step merged with the user Dockerfile (or working against the base image),
+	// the path will be the dev container context
+	// Set empty dir under temp path as the context for now to ensure we don't have dependencies on the features content
+	const emptyTempDir = path.posix.join(await cliHost.tmpdir(), '__dev-containers-build-empty');
```

Comment:
Should this use `cliHost.path` instead of `path.posix` to use the correct separator?